include/blmc_robots/solo12.hpp

```diff
@@ -206,7 +209,62 @@ class Solo12
         return slider_positions_;
     }
 
-    /**
+    /** @brief base accelerometer from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector3d> get_imu_accelerometer()
+    {
+        return imu_accelerometer_;
+    }
+
+    /** @brief base gyroscope from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector3d> get_imu_gyroscope()
+    {
+        return imu_gyroscope_;
+    }
+
+    /** @brief base attitude from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector3d> get_imu_attitude()
+    {
+        return imu_attitude_;
+    }
+
+    /** @brief base linear acceleration from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector3d> get_imu_linear_acceleration()
+    {
+        return imu_linear_acceleration_;
+    }
+
+    /** @brief base attitude quaternion from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector4d> get_imu_attitude_quaternion()
```

Comment:
Would it make sense to document the ordering of the quaternion in the 4 vector? E.g. is it `xyzw` or `wxyz` order.

Reply:
Done :+1: